anomalib/deploy/inferencers/base.py

```diff
@@ -22,11 +22,13 @@
 from omegaconf import DictConfig, OmegaConf
 from torch import Tensor
 
-from anomalib.data.utils import read_image
-from anomalib.utils.normalization.cdf import normalize as normalize_cdf
-from anomalib.utils.normalization.cdf import standardize
-from anomalib.utils.normalization.min_max import normalize as normalize_min_max
-from anomalib.utils.post_process import superimpose_anomaly_map
+from anomalib.post_processing.normalization.cdf import normalize as normalize_cdf
+from anomalib.post_processing.normalization.cdf import standardize
+from anomalib.post_processing.normalization.min_max import (
+    normalize as normalize_min_max,
+)
+from anomalib.post_processing.post_process import superimpose_anomaly_map
```

Comment:
Do you think if `anomalib.post_processing.post_process` look a bit verbose? Would it be an idea to do a relative import in __init__ such that we could import post_process functions via `from anomalib.post_processing import superimpose_anomaly_map`? What do you think?